bigquery/job.go

```diff
@@ -436,6 +436,10 @@ type QueryStatistics struct {
 	// statements INSERT, UPDATE or DELETE.
 	NumDMLAffectedRows int64
 
+	// DMLStatistics provides statistics about the row mutations performed by
```

Comment:
nit: DMLStatistics and DMLStats don't match. Either is fine by me.

Reply:
Matched the underlying pattern.  Object is now DMLStatistics, fieldname in querystatistics is DMLStats.